tests/pactHelper.js

```diff
@@ -513,6 +512,11 @@ function setGeneralInteractions (provider) {
   return promises
 }
 
+function pactCleanup (provider) {
```

Comment:
maybe call it `verifyAndCleanup`

Reply:
@individual-it can we do that after all the related PRs get merged, I'll get conflicts on all other PRs because of this

Reply:
:+1: